<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>login</class>
 <widget class="QWidget" name="login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>888</width>
    <height>619</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/*background-color:rgb(32, 102, 193);*/
background-color:#023047;</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>50</y>
     <width>401</width>
     <height>451</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white
;
border-radius:30px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget3">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>60</y>
     <width>381</width>
     <height>431</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white
;
border-radius:115px;</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>0</y>
       <width>161</width>
       <height>131</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="res.qrc">:/prefix/new/res/download (3).png</pixmap>
     </property>
    </widget>
    <widget class="QLineEdit" name="email">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>180</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
border:2px solid rgb(37,39,48);
border-radius:10px;

background-color:white;
}
QLineEdit:hover{
border:2px solid rgb(48,50,62);
}
QLineEdit:focus{
border:2px solid rgb(85,170,255);

}
</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>190</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
  position: relative;
  font-family: &quot;Avenir Next&quot;, sans-serif;
  font-weight:9000;
  font-size: 15px;
  text-transform: uppercase;
  font-style: italic;
  letter-spacing: 0.05em;
  display: inline-block;}
QLabel:Before{  position: absolute;
  left: 0;
  top: 0;
  content: attr(filter-content);

  filter: url(#motion-blur-filter);}</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;EMAIL:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="mdp">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>230</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::ClickFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
border:2px solid rgb(37,39,48);
border-radius:10px;

background-color:white;
}
QLineEdit:hover{
border:2px solid rgb(48,50,62);
}
QLineEdit:focus{
border:2px solid rgb(85,170,255);
}
</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>230</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
  position: relative;
  font-family: &quot;Avenir Next&quot;, sans-serif;
  font-weight:9000;
  font-size: 15px;
  text-transform: uppercase;
  font-style: italic;
  letter-spacing: 0.05em;
  display: inline-block;}
QLabel:Before{  position: absolute;
  left: 0;
  top: 0;
  content: attr(filter-content);

  filter: url(#motion-blur-filter);}</string>
     </property>
     <property name="text">
      <string>Password:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="valider">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>290</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
border-radius:15px;
background-color:#023047;
/*background-color:rgb(206, 212, 84);*/
color:rgb(255, 255, 255);
font: 75 12pt &quot;System&quot;;

}
QPushButton:hover{
background-color:#90e0ef;
font: 75 12pt &quot;System&quot;;
color:rgb(255, 255, 255);

}
QPushButton:pressed{
background-color:#90e0ef;
font: 75 12pt &quot;System&quot;;
color:white;

}</string>
     </property>
     <property name="text">
      <string>LOG IN</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_3">
    <widget class="QLineEdit" name="nom_nv">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>170</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
border:2px solid rgb(37,39,48);
border-radius:10px;

background-color:white;
}
QLineEdit:hover{
border:2px solid rgb(48,50,62);
}
QLineEdit:focus{
border:2px solid rgb(85,170,255);
}
</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="prenom_nv">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>210</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
border:2px solid rgb(37,39,48);
border-radius:10px;

background-color:white;
}
QLineEdit:hover{
border:2px solid rgb(48,50,62);
}
QLineEdit:focus{
border:2px solid rgb(85,170,255);
}
</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="adresse_nv">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>250</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
border:2px solid rgb(37,39,48);
border-radius:10px;

background-color:white;
}
QLineEdit:hover{
border:2px solid rgb(48,50,62);
}
QLineEdit:focus{
border:2px solid rgb(85,170,255);
}
</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="email_nv">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>290</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
border:2px solid rgb(37,39,48);
border-radius:10px;

background-color:white;
}
QLineEdit:hover{
border:2px solid rgb(48,50,62);
}
QLineEdit:focus{
border:2px solid rgb(85,170,255);
}
</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>340</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
  position: relative;
  font-family: &quot;Avenir Next&quot;, sans-serif;
  font-weight:9000;
  font-size: 15px;
  text-transform: uppercase;
  font-style: italic;
  letter-spacing: 0.05em;
  display: inline-block;}
QLabel:Before{  position: absolute;
  left: 0;
  top: 0;
  content: attr(filter-content);

  filter: url(#motion-blur-filter);}</string>
     </property>
     <property name="text">
      <string>MOT DE PASSE:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>180</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
  position: relative;
  font-family: &quot;Avenir Next&quot;, sans-serif;
  font-weight:9000;
  font-size: 15px;
  text-transform: uppercase;
  font-style: italic;
  letter-spacing: 0.05em;
  display: inline-block;}
QLabel:Before{  position: absolute;
  left: 0;
  top: 0;
  content: attr(filter-content);

  filter: url(#motion-blur-filter);}</string>
     </property>
     <property name="text">
      <string>NOM:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>260</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
  position: relative;
  font-family: &quot;Avenir Next&quot;, sans-serif;
  font-weight:9000;
  font-size: 15px;
  text-transform: uppercase;
  font-style: italic;
  letter-spacing: 0.05em;
  display: inline-block;}
QLabel:Before{  position: absolute;
  left: 0;
  top: 0;
  content: attr(filter-content);

  filter: url(#motion-blur-filter);}</string>
     </property>
     <property name="text">
      <string>ADRESSE:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>220</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
  position: relative;
  font-family: &quot;Avenir Next&quot;, sans-serif;
  font-weight:9000;
  font-size: 15px;
  text-transform: uppercase;
  font-style: italic;
  letter-spacing: 0.05em;
  display: inline-block;}
QLabel:Before{  position: absolute;
  left: 0;
  top: 0;
  content: attr(filter-content);

  filter: url(#motion-blur-filter);}</string>
     </property>
     <property name="text">
      <string>PRENOM:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>300</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
  position: relative;
  font-family: &quot;Avenir Next&quot;, sans-serif;
  font-weight:9000;
  font-size: 15px;
  text-transform: uppercase;
  font-style: italic;
  letter-spacing: 0.05em;
  display: inline-block;}
QLabel:Before{  position: absolute;
  left: 0;
  top: 0;
  content: attr(filter-content);

  filter: url(#motion-blur-filter);}</string>
     </property>
     <property name="text">
      <string>EMAIL:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="mdp_nv">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>330</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
border:2px solid rgb(37,39,48);
border-radius:10px;

background-color:white;
}
QLineEdit:hover{
border:2px solid rgb(48,50,62);
}
QLineEdit:focus{
border:2px solid rgb(85,170,255);
}
</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>-10</y>
       <width>101</width>
       <height>131</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="res.qrc">:/prefix/new/res/add.png</pixmap>
     </property>
    </widget>
    <widget class="QPushButton" name="valider_2">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>370</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
border-radius:15px;
background-color:#023047;
/*background-color:rgb(206, 212, 84);*/
color:rgb(255, 255, 255);
font: 75 12pt &quot;System&quot;;

}
QPushButton:hover{
background-color:#90e0ef;
font: 75 12pt &quot;System&quot;;
color:rgb(255, 255, 255);

}
QPushButton:pressed{
background-color:#90e0ef;
font: 75 12pt &quot;System&quot;;
color:white;

}</string>
     </property>
     <property name="text">
      <string>INSCRIT</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="id_nv">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>130</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
border:2px solid rgb(37,39,48);
border-radius:10px;

background-color:white;
}
QLineEdit:hover{
border:2px solid rgb(48,50,62);
}
QLineEdit:focus{
border:2px solid rgb(85,170,255);
}
</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>140</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
  position: relative;
  font-family: &quot;Avenir Next&quot;, sans-serif;
  font-weight:9000;
  font-size: 15px;
  text-transform: uppercase;
  font-style: italic;
  letter-spacing: 0.05em;
  display: inline-block;}
QLabel:Before{  position: absolute;
  left: 0;
  top: 0;
  content: attr(filter-content);

  filter: url(#motion-blur-filter);}</string>
     </property>
     <property name="text">
      <string>ID_EMPLOYER:</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>-50</x>
     <y>250</y>
     <width>331</width>
     <height>171</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="res.qrc">:/prefix/new/res/constop1-removebg-preview.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>311</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255);
font:&quot;System&quot;;
 animation: outer-left 1s 1s cubic-bezier(0.5, 0, 0.1, 1) both;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:48pt;&quot;&gt;Bienvenue!&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
